hoomd/md/pytest/test_rigid.py

```diff
@@ -453,3 +453,152 @@ def test_rigid_dof(
         thermo_central_free.translational_degrees_of_freedom == (n_bodies + n_free) * 3
     )
     assert thermo_constituent.translational_degrees_of_freedom == 0
+
+
+@pytest.mark.parametrize(
+    "com_velocity",
+    [(0, 0, 0), (1, 2, 3)],
+)
+def test_velocity_constituents(
```

Comment:
I think it's helpful to note these are constant torque, since the one below is `_constant_velocity`
```suggestion
def test_velocity_constituents_constant_torque(
```